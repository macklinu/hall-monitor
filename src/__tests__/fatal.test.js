import { createHallMonitor } from '..'

const hallMonitor = createHallMonitor({ parserOptions: { ecmaVersion: 3 } })

hallMonitor('fatal', () => ({}), {
  valid: ['var a'],
  invalid: ['const fn = () => {}'],
})
