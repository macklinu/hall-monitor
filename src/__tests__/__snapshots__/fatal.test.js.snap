// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fatal invalid const fn = () => {} 1`] = `[Error: Parsing error: Unexpected token fn]`;

exports[`fatal valid var a 1`] = `"var a"`;
